build/docker/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN         apk add --no-cache --upgrade \
                 mongodb-tools \
                 mysql-client \
                 postgresql-client \
+                redis \
```

Comment:
This will install a whole `redis-server`. Unfortunately there is no package like `redis-tools` which only installs `redis-cli` on alpine.

But copying only the `redis-cli`-binary from the official `redis:alpine`-image should be sufficient:

```Dockerfile
COPY --from=redis:alpine /usr/local/bin/redis-cli /usr/local/bin/redis-cli
```